statics/js/components/topology.js

```diff
@@ -226,6 +226,16 @@ var TopologyComponent = {
               :defaultKeys="[\'Last\', \'Start\', \'RxBytes\', \'RxPackets\', \'TxBytes\', \'TxPackets\']"></metrics-table>\
           </div>\
         </panel>\
+        <panel id="ovs-sflow-metric" v-if="currentNodeSFlowMetric"\
+               title="OVS SFlow Metrics">\
+           <h2>Total metrics</h2>\
+           <sflow-metrics-table :object="currentNodeSFlowMetric" :keys="globalVars[\'sflow-metric-keys\']"></sflow-metrics-table>\
+           <div v-show="currentNodeSFlowLastUpdateMetric && topologyTimeContext === 0">\
+             <h2>Last metrics</h2>\
+             <sflow-metrics-table :object="currentNodeSFlowLastUpdateMetric" :keys="globalVars[\'sflow-metric-keys\']" \
+                :defaultKeys="[\'Last\', \'IfInUcastPkts\', \'IfOutUcastPkts\', \'IfInMulticastPkts\', \'IfOutUcastPkts\', \'IfInDiscards\', \'IfOutDiscards\', \'IfInErrors\', \'IfOutErrors\']"></sflow-metrics-table>\
```

Comment:
could you remove multicast from the list as it is almost not used and trigger the horizontal scrollbar

Reply:
Ok!